README.md

```diff
@@ -6,7 +6,31 @@ This repository is cataloging University Library access to scholarly literature.
 Scholarly articles are identified using their DOIs.
 The impetus for this project was [this discussion](https://github.com/greenelab/scihub-manuscript/issues/21 "Potential followup: comparison to authorized access") on the Sci-Hub Coverage Study.
 
-## Environment
+The code in this repository facilitates fetching indicators of full-text availability for a list of DOIs from an OpenURL resolver. In this way, it enables large-scale analysis of bibliographic holdings / availability.
+
+## Using the Code
+
+**The code files in this repository assume that your working directory is set to the top-level directory of this repository.**
+
+### Contents of this Repository, and the Order of Their Use
+
+- `LICENSE-*.md`: License text to accompany the [License](#License) section of this Readme below.
+- `environment.yml`: Conda environment file (see [Environment](#environment) below).
+- `.gitattributes`: File with information for tracking files using [Git Large File Storage (LFS)](https://git-lfs.github.com/).
+- `library_management_system_downloader` contains the following scripts, to be used in the following order:
+	- `downloader_configuration_file_TEMPLATE.py` should be copied to `downloader_configuration_file.py` and edited for your own institution's OpenURL resolver (These scripts were specifically tested using the OpenURL resolver that comes with Ex Libris' Alma management software).
```

Comment:
For ordered things you may want to use a numbered list `1.`?

Reply:
Sure! Changed in b70710.